test/conftest.py

```diff
@@ -0,0 +1,39 @@
+import sys
```

Comment:
Where is this module used?

Reply:
The version check on line 4,5: https://github.com/cclib/cclib/pull/491/files#diff-f57b4d3a4403d70b647003618269deeaR4

Reply:
No, I mean where is `conftest` used :)

Reply:
I see. Could you add some comment to the file (perhaps in the docstring) explaining this?